src/index.html

```diff
@@ -1,17 +1,19 @@
 <!DOCTYPE html>
 <html lang="en">
+
```

Comment:
More of a standards question here, do you normally leave spaces between your elements in HTML?  I personally don't and haven't seen it but if you were told this is best practice I would love to hear about it.

Reply:
@ckoga Agh, it's something Atom Beautify does automatically.  It's not what Turing's rubric prefers so I will take it out.